### ISSUE TITLE
Glaze + Powertoys issue

### ISSUE THREAD

**Issue by monstajoe2002** [glzr-io/glazewm#663]
This happens only when powertoys is running in the background. GlazeWM thinks it's an actual window when it's not
![image](https://github.com/user-attachments/assets/ebaae6c9-25b3-44dd-bf67-27eff29684fa)

**Comment by ThePython10110** [comment]
Is this Powertoys Run? Or just some random white rectangle?

**Comment by monstajoe2002** [comment]
No, just some random white rectangle that appears whenever powertoys is running in the background

**Comment by lars-berger** [comment]
Haven't managed to reproduce this. Is it a specific Powertoys feature that's causing this? @monstajoe2002

**Comment by monstajoe2002** [comment]
It turns out it's caused by PowerAccent, but it's ignored in the config.yaml

**Comment by lars-berger** [comment]
> It turns out it's caused by PowerAccent, but it's ignored in the config.yaml

It still shows this white window despite the ignore window rule?

**Comment by monstajoe2002** [comment]
Yes

**Comment by lmcr666** [comment]
im also having problems with the 'ignore' command, i've added this window's process/name to the ignore window_rules but glazeWM doesnt really ignore it, sometimes it just resizes it to fit next to the opened window, others it just doesnt ignore the window, just tiles the window next to the other ones

**Comment by Jabbzz** [comment]
I managed to fix it. in the ignore section of the config file, i just put 'PowerToys', and it seems to currently be working
